### ISSUE TITLE
Admin notification email option

### ISSUE THREAD

**Issue by jjeaton** [jjeaton/bbpress-report-content#5]
Under consideration, but I'd like to avoid having a settings page for as long as possible.

**Comment by espellcaste** [comment]
Sorry, I think the day I posted the question above I was sleepy! 

Email notification to user reported would be interesting. Imagine if he/she post something forbidden but have no idea it is forbidden.

**Comment by jjeaton** [comment]
Just an email notification to the `admin_email` whenever any inappropriate content is reported. I just haven't got around to implementing that yet, as the initial use case for this plugin was on a very high traffic site. It's the most requested feature though.

**Comment by espellcaste** [comment]
Got it! What about the user reported? Like I said above, imagine I post something and someone report as inappropriate but I'm not aware it is? Would that be interesting too?

**Comment by mvaneijgen** [comment]
"It's the most requested feature though" yes it its. Only other option is [this plugin/repro](https://github.com/billerickson/bbPress-Report-Abuse/), but that requiers 'Gravity Forms'

**Comment by espellcaste** [comment]
@jjeaton Take a look at #13

**Comment by ghost** [comment]
I would have an email sent to admin + forum moderator(s)
Better still, instead of an email an overview in a dashboard with hyperlinks to to the topic reported

**Comment by espellcaste** [comment]
I disagree only on the last suggestion. Topic and reply reporting is not something believed to happen very often. So I guess maybe just an email would be enough.

**Comment by richrd** [comment]
This would be great. Any news on this?

Edit: I'll fork and implement it. I can do a PR too if you like.

**Comment by noreabu** [comment]
I stumbled upon this but need it very soon so i started coding.

I created a fork + branch from https://github.com/haleeben/bbpress-report-content with a feature that integrates the latest mail notifications and also adds the possibiliy to define a moderator mail address on a forum basis (+parents and global fallback)

If you are interested: https://github.com/noreabu/bbpress-report-content/tree/per-forum-mod

I will create a PR right away
